### PR TITLE
[BPK-4172] Upgrade storybook

### PR DESCRIPTION
**Pull request by georgegillams**
Remember to include the following changes:
+ [x] `UNRELEASED.yaml`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

**Review by olliecurtis — COMMENTED** [review]
LGTM! Just going to try running it locally and ensure all works as expected

**Review by olliecurtis — CHANGES_REQUESTED** [review]
Run locally and storybook loads fine with stories, however I noticed the `addons` section is not displaying even though I have it enabled so can't test accessibility, actions and knobs for components

Screenshot 2020-09-03 at 17 38 56

**Comment by georgegillams** [comment]
Good catch! I'll investigate!

File: package.json [hunk 1]
```diff
@@ -85,11 +85,11 @@
     "@babel/preset-react": "^7.8.3",
     "@babel/register": "^7.4.4",
     "@storybook/addon-a11y": "^5.3.14",
```
Comment (georgegillams) [inline package.json:87]: For some reason, the latest version get's stuck at "Initialising", and I can't find any related issues. For now I'll leave it here, and we can upgrade separately later.

**Review by olliecurtis — APPROVED** [review]
Tested locally and all seems to be working great!

**Pull request merged**